### PR TITLE
Ver 1.2

### PR DESCRIPTION
**Pull request by iamyajat**
## Update log
- Class Directions: Now get directions to your classes directly from the app!
- New taglines!
- Remote config added to control modes
- Minor bug fixes and improvements

## Repo updates
- README updated

### CODE REVIEW

**Review by rithikjain — CHANGES_REQUESTED** [review]
Everything else LGTM, @iamyajat just check through the one comment

File: app/build.gradle [hunk 1]
```diff
@@ -27,8 +27,8 @@ android {
         applicationId "com.dscvit.vitty"
         minSdkVersion 24
         targetSdkVersion 31
-        versionCode 3
-        versionName "1.1"
+        versionCode 14
+        versionName "1.2.0_beta7"
```
[the remark this answers is not in the document]
Comment (iamyajat) [reply app/build.gradle]: will change it to 1.2.0

[the remark this answers is not in the document]
Comment (rithikjain) [reply app/build.gradle]: Cool, will merge after that

[the remark this answers is not in the document]
Comment (iamyajat) [reply app/build.gradle]: done

**Review by rithikjain — APPROVED** [review]
LGTM

**Pull request merged**